.travis.rosinstall

```diff
@@ -1,29 +1,26 @@
-- git:
-   uri: http://github.com/RethinkRobotics/baxter_common.git
-   local-name: RethinkRobotics/baxter_common
-- git:
-    uri: https://github.com/jsk-ros-pkg/jsk_demos.git
-    local-name: jsk-ros-pkg/jsk_demos
-- git:
-    uri: https://github.com/jsk-ros-pkg/jsk_smart_apps.git
-    local-name: jsk-ros-pkg/jsk_smart_apps
-- git:
-    uri: https://github.com/jsk-ros-pkg/openrave_planning.git
-    local-name: jsk-ros-pkg/openrave_planning
-- git:
-    uri: https://github.com/code-iai/SnapMapICP.git
-    local-name: tum-ros-pkg/highlevel/SnapMapICP
+# - git:
```

Comment:
@k-okada This should not be removed but be moved to `.traivs.rosintall.hydro` until PR1040 is upgraded?

Reply:
Note: I'm working on another pull request and trying to test pr2 stuff both on hydro (from source) and on indigo (from apt).
https://github.com/jsk-ros-pkg/jsk_robot/pull/790/files#diff-354f30a63fb0907d4ad57269548329e3L26

Reply:
I found we do not need this for travis test. So If we need this, may be add them to pr2 related packages.

Reply:
@k-okada OK. moved to `.travis.rosinstall.hydro` and set up `.travis.yml` to test only pr2 related packages on hydro.
(Actually other packages like drc related packages always fails at this momemt.)